packages/backend/convex/http.ts

```
@@ -0,0 +1,77 @@
+    if (!event) {
+      return new Response('Error occurred', { status: 400 })
+    }
+
+    switch (event.type) {
+      case 'subscription.updated':
+        const subscription = event.data as {
+          status: string
+          payer?: {
+            organization_id?: string
+          }
+        }
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)